[docs] Introduce the Run Tests guide + refactor CLI and API (#4919)
DevExpress/testcafe: docs/articles/documentation/reference/api/runner/filter.md

---
layout: docs
title: Runner.filter Method
permalink: /documentation/reference/api/runner/filter.html
---
# Runner.filter Method

> Allows you to select which tests should be run.

Reviewer: ```suggestion
Allows you to select the tests to run.
```

```text
filter(callback) → this
```

Parameter  | Type                                                                  | Description
---------- | --------------------------------------------------------------------- | ----------------------------------------------------------------
`callback` | `function(testName, fixtureName, fixturePath, testMeta, fixtureMeta)` | The callback that determines if a particular test should be run.

> The callback function is called for each test in the files the [src](src.md) method specifies.

Reviewer: Maybe we can split this into two sentences? This one does not seem clear enough for the docs
